core/src/main/java/io/ran/Clazz.java

```diff
@@ -284,6 +284,10 @@ public KeySets getKeys() {
 		return fields;
 	}
 
+	public List<ClazzMethod> methods() {
+		return Arrays.stream(clazz.getMethods()).map(ClazzMethod::new).collect(Collectors.toList());
```

Comment:
I typically end up using "getDelcaredMethods()" - and need to traverse the hierarchy to get the proper ones. Otherwise I miss annotations on interface/superclass methods.

Reply:
see d616607